service/controller/app/v1/resource/tiller/resource.go

```diff
@@ -64,6 +64,14 @@ func (r *Resource) ensureTillerInstalled(ctx context.Context, helmClient helmcli
 		r.logger.LogCtx(ctx, "level", "debug", "message", "canceling reconciliation")
 		reconciliationcanceledcontext.SetCanceled(ctx)
 
+		return nil
+	} else if helmclient.IsTillerStartingError(err) {
```

Comment:
@rossf7 As we discussed in the previous PR, I added in here.

Reply:
```suggestion
	} else if helmclient.IsTillerNotRunningError(err) {
```

The logic is fine here but the naming I find confusing.

Pods do have a phase but it is running not started. I'd like us to align this with the upstream terminology.

```
kk get po tiller-deploy-584df9b797-n47kw -o yaml | grep phase
  phase: Running
```

Reply:
Sure I will change the helmclient library first then.